src/PDDoc.jl

```diff
@@ -129,13 +129,12 @@ Please refer to the PDF specification for further details.
 """
 function pdDocGetInfo(doc::PDDoc)
     obj = cosDocGetInfo(doc.cosDoc)
-    dInfo = Dict{CDTextString, Union{CDTextString, CDDate}}()
-    if obj != CosNull
-        for (key, val) in get(obj)
```

Comment:
Looks like your merge submission overwrote the earlier code. You may want to add it back.